engagement_database/backup-engagement-database.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+set -e
+
+if [[ $# -ne 7 ]]; then
+    echo "Usage: $0
+    <incremental-cache-volume> <google-cloud-credentials-file-path> <gcs-upload-prefix>
+    <engagement-database-credentials-file-url> <database-path>
+    <full-backup-interval-seconds> <incremental-backup-interval-seconds>
+    "
+    exit
+fi
+
+INCREMENTAL_CACHE_VOLUME=$1
+GOOGLE_CLOUD_CREDENTIALS_FILE_PATH=$2
+GCS_UPLOAD_PREFIX=$3
+ENGAGEMENT_DATABASE_CREDENTIALS_FILE_URL=$4
+DATABASE_PATH=$5
+FULL_BACKUP_INTERVAL=$6
+INCREMENTAL_BACKUP_INTERVAL=$7
+
+# Make sure there isn't a cache volume with this name already, so that the first backup runs correctly
+if docker volume inspect "$INCREMENTAL_CACHE_VOLUME" >/dev/null 2>&1; then
+    echo "Error: Volume '$INCREMENTAL_CACHE_VOLUME' already exists. Please remove it and re-run"
+    exit
+fi
+
+BACKUP_ID=$(uuidgen)
+LAST_FULL_BACKUP_SECONDS=0
+
+while true; do
+    # Check if enough time has passed since the last full backup, and set-up for a new full backup if it has.
+    if [ $(($(date -u +%s) - LAST_FULL_BACKUP_SECONDS)) -gt "$FULL_BACKUP_INTERVAL" ]; then
```

Comment:
I think LAST_FULL_BACKUP_SECONDS always be 0 here so `if [ $($(date -u +%s)) -gt "$FULL_BACKUP_INTERVAL" ];`is sufficient.